conanfile.txt

```diff
@@ -0,0 +1,13 @@
+[requires]
+cli11/[>=2.4.0 <2.5.0]
+spdlog/[>=1.14.0 <1.15.0]
+fmt/[>=10.0.0 <12.0.0]
+boost/[>=1.83.0 <1.84.0]
+magic_enum/[>=0.9.0 <0.10.0]
+gsl-lite/[>=0.41.0 <0.42.0]
+doctest/[>=2.4.0 <2.5.0]
+cryptopp/[>=8.9.0 <8.10.0]
+sdbus-cpp/[>=2.0.0 <2.1.0]
+
+[layout]
+cmake_layout
```

Comment:
I guess this file need to be deleted in favor of `connanfile.py`, right?
@viniciusferrao